.flowconfig

```diff
@@ -17,5 +17,4 @@
 
 [options]
 suppress_comment=.*\\$FlowFixMe
```

Comment:
You could also get rid of this line, since `$FlowFixMe` is suppressed by default, but that's not really related to this PR.